packages/protocol/README.md

```diff
@@ -26,43 +26,45 @@ If a new contract needs to be deployed, create a migration file in the `migratio
 To apply any new migrations to a network, run:
 
 ```bash
-yarn run migrate -n NETWORK
+yarn run devchain:migrate -n NETWORK
 ```
 
 ### Accounts
 
 To give an account some gold, wrapped gold, and stable token, run:
 
+[//]: <> (FIXME: faucet does not exist)
 ```bash
 yarn run faucet -n NETWORK -a ACCOUNT_ADDRESS
 ```
 
 You can check balances by running:
 
+[//]: <> (FIXME: get-balances does not exist)
```

Comment:
Same as above ☝️

Reply:
Instead there is https://docs.celo.org/cli/account#celocli-accountbalance-arg1 in celocli - this can be removed